src/ui/public/agg_types/buckets/histogram.js

```diff
@@ -47,7 +47,7 @@ define(function (require) {
           write: function (aggConfig, output) {
             var val = aggConfig.params.extended_bounds;
 
-            if (val.min != null || val.max != null) {
+            if (aggConfig.params.min_doc_count) {
```

Comment:
Can you explain your reasoning here? `min_doc_count` isn't required to be paired with `extended_bounds`. We could end up with null or undefined for `val.min` and `val.max`